src/Widgets/TerminalView.vala

```diff
@@ -65,12 +64,11 @@ public class Terminal.TerminalView : Gtk.Box {
         };
 
         tab_history_button = new Gtk.MenuButton () {
-            image = new Gtk.Image.from_icon_name ("document-open-recent-symbolic", Gtk.IconSize.MENU),
-            tooltip_text = _("Closed Tabs"),
-            use_popover = false,
```

Comment:
To fix this you can add `tab_history_button.popover.has_arrow = false;` right after `tab_history_button.menu_model = new Menu ();` on line 110

Reply:
Done